internal/logging/logging.go

```diff
@@ -0,0 +1,134 @@
+// Package logging implements the logger for the pack CLI.
```

Comment:
Is the intent that `pack` would move to start using this implementation by having `lifecycle` as a dependency so that there is no duplication (and maintenance overhead)?

Reply:
idk. maybe we split the logging stuff out as its only lib?

Reply:
@ekcasey what are your thoughts here?

Reply:
I considered the same thing way back when we began talking about a similar logger in lifecycle. I think a separate repo might have some overhead with maintenance, but not sure if I like pack having to continue to depend on lifecycle...

Reply:
I think we could consider this issue resolved. It seems like we are aligned on what the future would look like but not part of this issue.